### ISSUE TITLE
HTML native attributes are missing in tsx

### ISSUE THREAD

**Issue by laterdayi** [vuejs/language-tools#3301]
![image](https://github.com/vuejs/language-tools/assets/122137647/7a76ff01-441c-4912-9e53-f829905a7327)

```
        <n-tag type='default' onClick={() => test()}>
            test
        </n-tag>
```

**Comment by laterdayi** [comment]
```
<script setup lang="ts">
import { type TagProps } from 'naive-ui';

defineProps<TagProps>();
const slots = useSlots();
</script>

<template>
  <n-tag>

  </n-tag>
</template>

```

**Comment by laterdayi** [comment]
It is OK to use it in vue templates. But a type error is displayed in tsx

**Comment by yx1126** [comment]
I have the same problem
This is my menu component

![image](https://github.com/vuejs/language-tools/assets/59458861/c91a8e4e-3e6e-4c3b-b49e-7635c06bced5)

and when i used it in the tsx file 

![image](https://github.com/vuejs/language-tools/assets/59458861/a93da439-9846-42b3-95f8-749edcdcf545)
![image](https://github.com/vuejs/language-tools/assets/59458861/b3e58dd2-8cdf-4ce5-92a2-a9e307d3b00c)

**Comment by laterdayi** [comment]
```
<script setup lang="tsx">



const render = () => {
  return (
    <>
      <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" fill="none">
        <defs>
          <path id="path_0" transform="translate(0 0) rotate(0 44 30)" d="M0,60L88,60L88,0L0,0L0,60Z" />
          <filter
            id="filter_13"
            x="0"
            y="0"
            width="44"
            height="104"
            filterUnits="userSpaceOnUse"
            color-interpolation-filters="sRGB"
          >
            <feFlood flood-opacity="0" result="BackgroundImageFix" />
            <feBlend mode="normal" in="SourceGraphic" in2="BackgroundImageFix" result="shape" />
            <feGaussianBlur stdDeviation="25" result="effect1_foregroundBlur" />
          </filter>
          <filter
            id="filter_15"
            x="0"
            y="0"
            width="180"
            height="44"
            filterUnits="userSpaceOnUse"
            color-interpolation-filters="sRGB"
          >
            <feFlood flood-opacity="0" result="BackgroundImageFix" />
            <feBlend mode="normal" in="SourceGraphic" in2="BackgroundImageFix" result="shape" />
            <feGaussianBlur stdDeviation="25" result="effect1_foregroundBlur" />
          </filter>
          <path
            id="path_1"
            transform="translate(0 0) rotate(0 44 44)"
            d="M40,2.31L9.89,19.69C7.41,21.12 5.89,23.76 5.89,26.62L5.89,61.38C5.89,64.24 7.41,66.88 9.89,68.31L40,85.69C42.48,87.12 45.52,87.12 48,85.69L78.11,68.31C80.59,66.88 82.11,64.24 82.11,61.38L82.11,26.62C82.11,23.76 80.59,21.12 78.11,19.69L48,2.31C45.52,0.88 42.48,0.88 40,2.31Z"
          />
          <filter
            id="filter_41"
            x="0"
            y="0"
            width="0"
            height="166"
            filterUnits="userSpaceOnUse"
            color-interpolation-filters="sRGB"
          >
            <feFlood flood-opacity="0" result="BackgroundImageFix" />
            <feBlend mode="normal" in="SourceGraphic" in2="BackgroundImageFix" result="shape" />
            <feGaussianBlur stdDeviation="3" result="effect1_foregroundBlur" />
          </filter>
        </defs>
        <g opacity="1" transform="translate(0 0)  rotate(0 100 70)">
          <g opacity="1" transform="translate(30 7)  rotate(0 70 60)">
            <g opacity="1" transform="translate(0 60)  rotate(0 44 30)">
              <g opacity="1" transform="translate(0 0)  rotate(0 44 30)">
                <mask id="mask-0" fill="white">
                  <use xlink:href="#path_0" />
                </mask>
                <g mask="url(#mask-0)">
                  <path
                    id="Rectangle 4232"
                    fill-rule="evenodd"
                    style={{ fill: `#97a3b7` }}
                    transform="translate(0 0)  rotate(0 44 30)"
                    opacity="1"
                    d="M0,60L88,60L88,0L0,0L0,60Z "
                  />
                </g>
                <g mask="url(#mask-0)">
                  <g id="Rectangle 4227" filter="url(#filter_13)">
                    <path
                      id="Rectangle 4227"
                      fill-rule="evenodd"
                      style={{ fill: `#e3e6eb` }}
                      transform="translate(-18 22)  rotate(0 40 30)"
                      opacity="1"
                      d="M0,60L80,60L80,0L0,0L0,60Z "
                    />
                  </g>
                </g>
                <g mask="url(#mask-0)">
                  <g id="Rectangle 4228" filter="url(#filter_15)">
                    <path
                      id="Rectangle 4228"
                      fill-rule="evenodd"
                      style={{ fill: `#e3e6eb` }}
                      transform="translate(50 -8)  rotate(0 40 30)"
                      opacity="1"
                      d="M0,60L80,60L80,0L0,0L0,60Z "
                    />
                  </g>
                </g>
              </g>
              <g opacity="1" transform="translate(16 27)  rotate(0 20 8.5)">
                <path id="Vector 67" transform="translate(0 0)  rotate(0 9 0)" d="M0,0L18,0 " />
                <path id="Vector 68" transform="translate(0 10)  rotate(0 20 0)" d="M0,0L40,0 " />
                <path id="Vector 69" transform="translate(0 17)  rotate(0 20 0)" d="M0,0L40,0 " />
              </g>
            </g>
            <g opacity="1" transform="translate(39 0)  rotate(0 46.5 44)">
              <g opacity="1" transform="translate(2 0)  rotate(0 44 44)">
                <g opacity="1" transform="translate(0 0)  rotate(0 44 44)">
                  <g opacity="1" transform="translate(0 0)  rotate(0 44 44)">
                    <mask id="mask-1" fill="white">
                      <use xlink:href="#path_1" />
                    </mask>
                    <g mask="url(#mask-1)">
                      <path
                        id="Polygon 6"
                        fill-rule="evenodd"
                        transform="translate(0 0)  rotate(0 44 44)"
                        opacity="1"
                        d="M40,2.31L9.89,19.69C7.41,21.12 5.89,23.76 5.89,26.62L5.89,61.38C5.89,64.24 7.41,66.88 9.89,68.31L40,85.69C42.48,87.12 45.52,87.12 48,85.69L78.11,68.31C80.59,66.88 82.11,64.24 82.11,61.38L82.11,26.62C82.11,23.76 80.59,21.12 78.11,19.69L48,2.31C45.52,0.88 42.48,0.88 40,2.31Z "
                      />
                    </g>
                    <g mask="url(#mask-1)">
                      <path
                        id="Polygon 5"
                        fill-rule="evenodd"
                        transform="translate(0 -44)  rotate(0 44 44)"
                        opacity="1"
                        d="M40,2.31L9.89,19.69C7.41,21.12 5.89,23.76 5.89,26.62L5.89,61.38C5.89,64.24 7.41,66.88 9.89,68.31L40,85.69C42.48,87.12 45.52,87.12 48,85.69L78.11,68.31C80.59,66.88 82.11,64.24 82.11,61.38L82.11,26.62C82.11,23.76 80.59,21.12 78.11,19.69L48,2.31C45.52,0.88 42.48,0.88 40,2.31Z "
                      />
                    </g>
                    <g mask="url(#mask-1)">
                      <path
                        id="路径 2"
                        transform="translate(36 27.2)  rotate(0 7.33688756096788 5.137343977186761)"
                        d="M0,0L14.67,10.27 "
                      />
                    </g>
                    <g mask="url(#mask-1)">
                      <path
                        id="路径 3"
                        transform="translate(36 27.2)  rotate(0 7.336887560967881 5.1373439771867595)"
                        d="M14.67,0L0,10.27 "
                      />
                    </g>
                    <g mask="url(#mask-1)">
                      <g id="Polygon 7" filter="url(#filter_41)">
                        <path
                          id="Polygon 7"
                          fill-rule="evenodd"
                          transform="translate(-51 39)  rotate(0 44 44)"
                          opacity="0.3"
                          d="M40,2.31L9.89,19.69C7.41,21.12 5.89,23.76 5.89,26.62L5.89,61.38C5.89,64.24 7.41,66.88 9.89,68.31L40,85.69C42.48,87.12 45.52,87.12 48,85.69L78.11,68.31C80.59,66.88 82.11,64.24 82.11,61.38L82.11,26.62C82.11,23.76 80.59,21.12 78.11,19.69L48,2.31C45.52,0.88 42.48,0.88 40,2.31Z "
                        />
                      </g>
                    </g>
                  </g>
                </g>
              </g>
              <path
                id="路径 1"
                fill-rule="evenodd"
                transform="translate(45 22)  rotate(0 23.999999999999996 17.500000000000004)"
                opacity="1"
                d="M46.74,13.2L37.5,0L0,20.5L10.84,32.59C12.1,34 14.17,34.34 15.81,33.4L46.1,16.09C46.59,15.81 46.94,15.34 47.06,14.79C47.18,14.24 47.07,13.66 46.74,13.2Z "
              />
              <path
                id="路径 1"
                fill-rule="evenodd"
                transform="translate(0 21)  rotate(0 22.5 17.500000000000004)"
                opacity="1"
                d="M45,21.5L10,0L1.29,11.33C0.95,11.76 0.81,12.32 0.9,12.86C0.98,13.4 1.29,13.89 1.74,14.2L29.21,33.08C30.82,34.18 32.99,33.96 34.35,32.56L45,21.5Z "
              />
              <path
                id="路径 2"
                fill-rule="evenodd"
                transform="translate(5 1)  rotate(0 20.5 10)"
                opacity="1"
                d="M39.25,0.25L37.5,0.5L34.55,0.92C34.18,0.97 33.83,1.09 33.51,1.27L1.68,19.06C1.48,19.17 1.38,19.4 1.43,19.62C1.49,19.85 1.69,20 1.92,20L5.5,20L41,0L39.25,0.25Z "
              />
              <path
                id="路径 2"
                fill-rule="evenodd"
                transform="translate(46 1)  rotate(0 20.750000000000004 10.5)"
                opacity="1"
                d="M36.5,21L39.69,21C39.91,21 40.11,20.85 40.17,20.63C40.23,20.41 40.13,20.18 39.94,20.07L8.52,1.31C8.18,1.1 7.8,0.97 7.4,0.91L4.5,0.5L2.75,0.25L0,0L36.5,21Z "
              />
            </g>
            <g opacity="1" transform="translate(96 66)  rotate(0 22 22)">
              <path
                id="Ellipse 1"
                fill-rule="evenodd"
                style={{ fill: `#fff` }}
                transform="translate(0 0)  rotate(360 22 22)"
                opacity="1"
                d="M22,44C34.15,44 44,34.15 44,22C44,9.85 34.15,0 22,0C9.85,0 0,9.85 0,22C0,34.15 9.85,44 22,44Z "
              />
              <path
                id="Ellipse 1"
                transform="translate(0 0)  rotate(360 22 22)"
                d="M22,44C34.15,44 44,34.15 44,22C44,9.85 34.15,0 22,0C9.85,0 0,9.85 0,22C0,34.15 9.85,44 22,44Z "
              />
              <rect
                id="Rectangle 5"
                style={{ stroke: `#181818`, strokeWidth: `2`, strokeDasharray: `0 0` }}
                transform="translate(22 31) rotate(0 0 0)"
                x="1"
                y="1"
                rx="0"
                width="0"
                height="2"
              />
            </g>
          </g>
        </g>
      </svg>
    </>
  );
};


</script>

<template>
  <button></button>
</template>
```

**Comment by laterdayi** [comment]
不能将类型“{ xmlns: string; "xmlns:xlink": string; fill: string; }”分配给类型“SVGAttributes & ReservedProps”。
  类型“SVGAttributes & ReservedProps”上不存在属性“xmlns:xlink”。

**Comment by so1ve** [comment]
Closed because #3496 is more descriptive
